modules/check_in/app/sidekiq/check_in/travel_claim_status_check_worker.rb

```diff
@@ -46,36 +51,84 @@ def claim_status(opts = {})
 
     # rubocop:disable Metrics/MethodLength
     def handle_response(opts = {})
-      # claim_response_body = opts[:claim_status_resp].body || []
+      claim_response_body = opts[:claim_status_resp].body || []
       claim_response_status = opts[:claim_status_resp].status
       facility_type = opts[:facility_type] || ''
       claim_number = nil
 
       statsd_metric, template_id = case claim_response_status
```

Comment:
I'd suggest to move this to a separate method.

Reply:
Please refer to the latest commit